src/components/Sidebar/Sidebar.jsx

```diff
@@ -32,7 +32,8 @@ export default class Sidebar extends Component {
           <SidebarItem
             url={ `/${sectionName}/` }
             title="Introduction"
-            currentPage= { currentPage } />
+            currentPage= { currentPage }
+            anchors= { anchors } />
```

Comment:
Yeah, this works for now. Ideally we should just treat index pages as a normal page so they are handled by `pages.map` making this extra item unnecessary. I'm ok with this for now though, just something to consider in the future.